go.mod

```diff
@@ -3,55 +3,20 @@ module github.com/kubeflow/tf-operator
 go 1.14
 
 require (
-	github.com/coreos/etcd v3.3.20+incompatible // indirect
-	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
 	github.com/go-openapi/spec v0.19.2
-	github.com/golang/protobuf v1.3.2 // indirect
 	github.com/google/go-cmp v0.4.1 // indirect
-	github.com/grpc-ecosystem/grpc-gateway v1.5.0 // indirect
-	github.com/kubeflow/common v0.3.3-0.20210227095238-97658773cce1
+	github.com/kubeflow/common v0.3.3
```

Comment:
/cc @Jeffwan 

Should we use the new version of common now?

Reply:
IMO, for the current version 1.16, we can accept this change and issue a version, and then upgrade to 1.19 in another pr if necessary.

@gaocegege @Jeffwan WDYT>?

Reply:
Yeah, 0.3.4 bump dependency to 1.19 which brings breaking changes. We can have it as a separate PR.

Reply:
em. I notice this PR only have upgrade version. In that case, I actually suggest to bump to latest version directly.

Reply:
If we use common@0.3.4, the k8s related dependency will automatically rise to 1.19, and need to use replace to downgrade to 1.16, l think it’s better to release one for version 1.16, WDYT?

Reply:
If the goal is to upgrade volcano version from 0.4.0 to 1.2.0, I think it makes sense to have two separate PRs. Upgrade kubernetes client actually is not a problem.